ansible_collections/roles/haproxy/defaults/main.yml

```diff
@@ -0,0 +1,54 @@
+haproxy_exporter_version: 0.10.0
+# Default haproxy config
+haproxy_config_template: haproxy.cfg.j2
+# Backend haproxy config
+#haproxy_config_template: backend-haproxy.cfg.j2
+install_haproxy_tools: true
```

Comment:
I would name the variables in a way that you can see that they belong to haproxy (eg haproxy_). When you use them later with variables from other roles than it can be confusing when you dont know which role they belong to.